ir/visitor.cpp

```diff
@@ -27,44 +35,60 @@ class Visitor::ChangeTracker {
     visited_t           visited;
 
  public:
-
-    /* Begin tracking `n` during a visiting pass.  Use `finish` to mark `n` as
+    /** Begin tracking @n during a visiting pass.  Use `finish(@n)` to mark @n as
      * visited once the pass completes.
-    */
+     */
     void start(const IR::Node *n) {
         // Initialization
         visited_t::iterator visited_it;
         bool inserted;
         std::tie(visited_it, inserted) =
-            visited.emplace(n, visit_info_t({true,n}));
+            visited.emplace(
+                n,
+                visit_info_t{.visit_in_progress = true, .result = n});
```

Comment:
I'm sorry to say it, but designated initializers are not C++11. =\ We can't use these.